.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -34,6 +36,17 @@ body:
         3. out
     validations:
       required: false
+  - type: checkboxes
+    attributes:
+      label: Package Version
+      description: Confirm if you are using the latest version of iTwinUI.
+      options:
+        - label: I'm using the latest version of iTwinUI.
+          required: false
+        - label: I'm using other version of iTwinUI.
+          required: false
+    validations:
+      required: true
```

I was thinking a single checkbox should be enough. If it is unchecked, it implies they are using a different version. Also we should include a link so that they know what the latest version is.

```suggestion
  - type: checkboxes
    attributes:
      label: Package Version
      options:
        - label: I'm using the [latest version](https://www.npmjs.com/package/@itwin/itwinui-react?activeTab=versions) of iTwinUI.
          required: false
    validations:
      required: false
```

---

I'm not sure why the link markdown is not supported in `label` for checkbox `option`. I have looked around, but I haven't found a way to do as suggested. I have added a description to include the link in your suggestion but not sure this is okay.

---

Yeah I guess this is a limitation in Github, nothing we can do about it.